ui/StatusQ/src/StatusQ/Components/StatusStepper.qml

```diff
@@ -0,0 +1,86 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+/*!
+   \qmltype StatusStepper
+   \inherits Item
+   \inqmlmodule StatusQ.Components
+   \since StatusQ.Components 0.1
+   \brief Displays total number of steps which need to be passed, marking each completed step
+          based on `completedSteps` property
+
+   Example:
+
+   \qml
+    StatusStepper {
+        width: 400
+        title: "Account %1 of %2"
+        totalSteps: 6
+        completedSteps: 1
+    }
+   \endqml
+
+   \image status_stepper.png
+
+   For a list of components available see StatusQ.
+*/
+
+Item {
+    id: root
+
+    property string title: ""
```

Comment:
Will a reminder comment that this expects a formatted string with `%1` and `%2` help?

```suggestion
    /// Expected formatted text with %1 place marker for completed steps and %2 place marker for total steps count
    property string title: ""
```

Reply:
Ouch, smells like a gross hack :) Why not have the string right here, e.g.:
```suggestion
    property string title: qsTr("Account %1 of %2").arg(completedSteps).arg(totalSteps)
```

Reply:
Ah I see... because the actual text can change?